### PR TITLE
Feature: Update packaging backend to `poetry`

### PR DESCRIPTION
**Pull request by umoqnier**
Currently, we use `setuptools` as the mechanism for packaging `py-elotl`. I think that updating this packaging backend to a modern one like [poetry](https://python-poetry.org/docs/) can be convenient in terms of maintainability. 

Poetry can be used for:

- Better dependency resolution, in contrast with the old fashion `pip install -r requirements.txt`
- Virtualenvs management
- Built-in building and packaging
- Publishing to PyPi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 Developed by:
 - Paul Aguilar [@penserbjorne](https://github.com/penserbjorne), [paul.aguilar.enriquez@hotmail.com](mailto:paul.aguilar.enriquez@hotmail.com)
 - Robert Pugh [@Lguyogiro](https://github.com/Lguyogiro), [robertpugh408@gmail.com](mailto:robertpugh408@gmail.com)
+- Diego Barriga [@umoqnier](https://github.com/umoqnier/), [dbarriga@ciencias.unam.mx](mailto:dbarriga@ciencias.unam.mx)
 
-Requiere python>=3.X
+Requiere python>=3.6
 
-- Development Status `Pre-Alpha`. Read [Classifiers](https://pypi.org/classifiers/)
+- Development Status `Alpha`. Read [Classifiers](https://pypi.org/classifiers/)
```
Comment (umoqnier) [inline README.md:15]: I think that the project is mature enough to change the status to `Alpha`. What do you think?

File: README.md [hunk 2]
```diff
@@ -96,30 +101,42 @@ tsunkua = elotl.corpus.load('tsunkua')
...
+├── dist
+├── docs
+├── elotl                           Top-level package
+    ├── corpora                     Here are the corpus data
+    ├── corpus                      Subpackage to load corpus
+    ├── huave
+    ├── __init__.py                 Initialize the package
+    ├── nahuatl                     Nahuatl language subpackage
+        └── orthography.py          Module to normalyze nahuatl orthography and phonemas
+    ├── otomi
```
Comment (umoqnier) [inline README.md:129]: Feel free to update missing desc

File: README.md [hunk 3]
```diff
@@ -129,19 +146,19 @@ test/                               Unit test scripts
...
-  - wheel
+- [poetry](https://python-poetry.org/docs/)
+    - For python packaging backend and virtualenvs
 
 ### Quick build
 
 ```bash
-virtualenv --python=/usr/bin/python3 venv
-source venv/bin/activate
+poetry env use 3.x
```
Comment (umoqnier) [inline README.md:155]: image

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "elotl"
+version = "0.0.17"
+description = "Python package for Natural Language Processing (NLP), focused on low-resource languages spoken in Mexico."
+authors = [
+	"Paul Aguilar <paul.aguilar.enriquez@hotmail.com>",
+	"Robert Pugh <robertpugh408@gmail.com>",
+	"Diego A. Barriga Martínez <dbarriga@ciencias.unam.mx>"
+]
+maintainers = ["Robert Pugh <robertpugh408@gmail.com>"]
```
Comment (umoqnier) [inline pyproject.toml:10]: I'm not sure about this. Is it correct?

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "elotl"
+version = "0.0.17"
+description = "Python package for Natural Language Processing (NLP), focused on low-resource languages spoken in Mexico."
+authors = [
+	"Paul Aguilar <paul.aguilar.enriquez@hotmail.com>",
+	"Robert Pugh <robertpugh408@gmail.com>",
+	"Diego A. Barriga Martínez <dbarriga@ciencias.unam.mx>"
```
Comment (umoqnier) [inline pyproject.toml:8]: I took the liberty of adding myself to the list :paintbrush:

File: README.md (hunk 1 above)
Comment (Lguyogiro) [reply README.md:15]: yes, this sounds good to me.

File: README.md (hunk 2 above)
Comment (Lguyogiro) [reply README.md:129]: I think this is mostly correct. The only thing (I think) missing is that huave, nahuatl, and otomi all have a `morphology.py` Module for morphological analysis.

File: pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+]
+maintainers = ["Robert Pugh <robertpugh408@gmail.com>"]
+license = "MLP-2.0"
+readme = "README.md"
+repository = "https://github.com/umoqnier/elotl_pkg"
```
Comment (Lguyogiro) [inline pyproject.toml:13]: Is this supposed to point to the fork?

**Review by Lguyogiro — APPROVED** [review]
LGTM

File: pyproject.toml (hunk 6 above)
Comment (umoqnier) [reply pyproject.toml:13]: Oh no, thanks. I will update this. Sorry :p

**Review by penserbjorne — APPROVED** [review]
Hello!

I'm just making this comment because a review from me was requested but I'm not longer active in the maintenance of the package so I don't want to stop the development of it.

From my perspective everything seems good, I think Robert made the right comments so just fix that. 

Also, let me know what is the best way to give permissions to the new maintainers on pipy so you can updated or do new releases.

Best,
P

File: pyproject.toml (hunk 6 above)
Comment (umoqnier) [reply pyproject.toml]: Done!

**Pull request merged**